inc/saidot1brfdb.h

```diff
@@ -0,0 +1,130 @@
+/*
+* Copyright (c) 2015 Dell Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may
+* not use this file except in compliance with the License. You may obtain
+* a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR
+* CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT
+* LIMITATION ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS
+* FOR A PARTICULAR PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+*
+* See the Apache Version 2.0 License for specific language governing
+* permissions and limitations under the License.
+*
+* @file saidot1brfdb.h
+*
+* @brief This module defines SAI API for IEEE 802.1BR ECID based forwarding
+         functionality. This is applicable only to Port Extenders (PE).
+*************************************************************************************/
+
+#if !defined (__SAIDOT1BRFDB_H)
+#define __SAIDOT1BRFDB_H
+
+#include "saitypes.h"
+#include "saistatus.h"
+
+/** \defgroup SAIDOT1BRFDBENTRY SAI - 802.1BR ECID based forwarding specific public APIs and datastructures in PE.
+ *
+ *  \{
+ */
+
+/**
+ * @brief SAI attributes for SAI_OBJECT_TYPE_DOT1BR_FDB_ENTRY 
+ */
+typedef enum _sai_dot1br_fdb_entry_attr_t
```

Comment:
Cant this be merged with the existing fdb object. Can have an attribute to mark egress as BR port/list or if the BR port list is non null, the FDB points to BR list.